include/zephyr/pm/policy.h

```diff
@@ -48,6 +48,13 @@ struct pm_policy_latency_request {
 	uint32_t value;
 };
 
+/** @brief Event. */
+struct pm_policy_event {
+	sys_snode_t node;
+	/** Request value. */
+	uint32_t value;
```

Comment:
value is quite confuse IMHO. Can we make it more explicit like `cycs_to_event` or something that says what it is representing ?

Reply:
Note this is an internal value, but we can maybe adjust its name explicitly mentioning this is for internal use.